### PR TITLE
Move user login for presenter

### PR DESCRIPTION
**Pull request by jsangmeister**

### CODE REVIEW

**Comment by jsangmeister** [comment]
waiting for auth service update from @GabrielInTheWorld

File: tests/unit/test_presenter.py [hunk 1]
```diff
@@ -16,12 +16,10 @@ def setUp(self) -> None:
         self.user_id = 0
 
     def test_with_bad_key(self) -> None:
```
Comment (FinnStutzenstein) [inline tests/unit/test_presenter.py:18]: Please add a test for two presenters with different auth modes

Comment (jsangmeister) [reply tests/unit/test_presenter.py:18]: done

**Comment by GabrielInTheWorld** [comment]
> waiting for auth service update from @GabrielInTheWorld

@jsangmeister: I have updated the pip-library of the auth-service. So, the backend can validate an authentication only with a jwt-cookie for mediafiles.

**Comment by jsangmeister** [comment]
Should be done now

**Review by FinnStutzenstein — APPROVED** [review]

**Pull request merged**